### DISCUSSION TITLE
Menu "PROC"

### DISCUSSION THREAD

**Discussion by 1972igor** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#21] [Ideas]
ESDR3 v 0.10.0
Допускаю, что можно использовать «Микрофонный метр» для мониторинга каждого модуля «ПРОК», но думаю, что этот индикатор устанавливается на входе модулятора, после внутреннего АРУ передатчика, АРУ передатчика работает после 0дБ! И мы не увидим, какой модуль обработки имеет уровень выше 0 дБ!! Но сегодня индикатор Mic meter дает нам неверную информацию о входном уровне, а именно он не работает линейно на нижних проходах!!! нужны точные измерители уровня для каждого модуля / этапа в меню proc.
OdB сигнала на микрофонном входе должен показывать 0 дБ на метрах, он также должен быть линейным и корректным считываться на более низких уровнях. Измерение для каждого этапа обработки поможет пользователю поддерживать правильный уровень усиления и удерживать от чрезмерного или недостаточного вождения P.A в радио.
![image](https://user-images.githubusercontent.com/93991940/144988350-944d7488-6543-4800-a219-29dd47c7f672.png)

![image](https://user-images.githubusercontent.com/93991940/144988505-c96f1b4f-a88d-4968-bca4-fbacbeb8811f.png)
![bandicam 2021-12-07 10-39-47-797](https://user-images.githubusercontent.com/93991940/144988815-a7cb5d8e-346b-4a2f-895e-f91ec00e97fe.jpg)
![bandicam 2021-12-07 10-54-22-004](https://user-images.githubusercontent.com/93991940/144988853-5d94e346-c54d-4145-8883-56b8d6ccaf7c.jpg)
![bandicam 2021-12-07 10-39-22-282](https://user-images.githubusercontent.com/93991940/144988899-bc0b28f8-8201-462d-bff3-ab0ddf751267.jpg)
![bandicam 2021-12-07 10-39-30-947](https://user-images.githubusercontent.com/93991940/144988902-1b9d03c1-93af-4c5a-bf37-e3ac46f75188.jpg)
![bandicam 2021-12-07 10-39-47-797](https://user-images.githubusercontent.com/93991940/144988903-8d394b39-7153-4756-932c-8dbaea7d723a.jpg)
![bandicam 2021-12-07 10-53-55-003](https://user-images.githubusercontent.com/93991940/144988904-07f1b6fa-3b90-41f5-a703-1c2390085eaa.jpg)
![bandicam 2021-12-07 10-54-03-258](https://user-images.githubusercontent.com/93991940/144988905-55e1e407-6802-4b1c-808a-c66487646441.jpg)

**Comment by VK6NX** [comment]
**_English translation_**

I would propose to use "Mic Meter" to monitor PROC modules. The indication can be taken from modulator input and after internal AGC (as the AGC works at 0dB, then we cannot detect correctly which module creates above 0dB level). In current implementation the source of incorrect "IN level" is provided by Mic Indicator, because of Mic Indicator does not work on low levels. To fix this we need precise measurements of the each module "In" and "Out" levels and the "PROC menu" seems to be reasonable location of those measurements.

The 0dB signal level (from Mic Input) should be reflected with 0dB indication on all responsible/correlated meters. The 0dB level should also have linear and correct readings on low signal levels. The precise level of measurements at every stage of signal processing will help to keep correct attenuation level; it also prevent the PA from over or under load.

**Comment by VK6NX** [comment]
IMHO, the indications of staged measurements has to be present in "basic" ESDR3 - as minimum. Better - with giving user the ability to perform minimal adjustments to line up real 0dB level (of his input environment) with indicated one.

For "pro" ESDR3 the extended correction/settings need to be implemented (what exactly - this need to be discussed separately).

**Reply by N8SDR007** [reply]
I agree 100% this has to come in the very BASIC version of EESDR3- and with more details in the later "PRO" version of EESDR3

**Reply by Kb2uka** [reply]
That would make a world of difference, the ability to bring the level to a true zero db.

**Comment by N8SDR007** [comment]
Curious in this discussion are we talking Consumer line level (-10 dBv) or roughly 326millivolts or using professional equipment levels of (+4 dBu) almost 1.23 volts as that will have impact on input gain structures and correct readings from the start, I ask because if one was bypassing the internal processing available and used say rack gear the incorrect values for input would greatly affect the overall outcome. So, a baseline should in my opinion be stated as to which is being used otherwise were talking around a -12dB to low or a +12dB too hot, 

I think I have that correct?

![diagram-two-768x354](https://user-images.githubusercontent.com/94698375/145125639-901405c0-1388-4808-99be-a4ea5ef34b88.jpg)
